File: Practice/AllDomains/Tutorials/30DaysOfCode/Day2Operators.java
public class Arithmetic 
{
	/*	author:@shivkrthakur  */
    public static void main(String[] args) 
	{
        Scanner scan = new Scanner(System.in);
        double mealCost = scan.nextDouble(); // original meal price
        int tipPercent = scan.nextInt(); // tip percentage
        int taxPercent = scan.nextInt(); // tax percentage
        scan.close();
      
        double tip = (tipPercent/100.0) * mealCost;
        double tax = (taxPercent/100.0) * mealCost;
        
        // Write your calculation code here.
      
        // cast the result of the rounding operation to an int and save it as totalCost 
        int totalCost = (int) Math.round(mealCost + tip + tax);
        System.out.println("The total meal cost is " + totalCost + " dollars.");
        // Print your result
    }
}

File: Practice/AllDomains/Tutorials/30DaysOfCode/Day2Operators.cs
using System;
using System.Collections.Generic;
using System.IO;
class Solution 
{
	/*	author:@shivkrthakur  */
    static void Main(String[] args) 
	{
        /* Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution */
        double mealCost = Convert.ToDouble(Console.ReadLine().Trim());
        double tipPerc = Convert.ToDouble(Console.ReadLine().Trim());
        double taxPerc = Convert.ToDouble(Console.ReadLine().Trim());
        
        double tip = ((tipPerc/100) * mealCost);
        double tax = ((taxPerc/100) * mealCost);
        
        int totalMealCost = Convert.ToInt32(mealCost + tip + tax);
        Console.WriteLine("The total meal cost is {0} dollars.", totalMealCost);
    }
}